SportsPlanner.iOS/SportsPlanner.iOS.csproj.user

```diff
@@ -2,5 +2,6 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Maybe we can avoid the hardcoded strings by `nameOf`
https://msdn.microsoft.com/en-us/library/dn986596.aspx